src/connectors/numeric-refinement-list/connectNumericRefinementList.js

```diff
@@ -173,6 +172,85 @@ export default function connectNumericRefinementList(renderFn, unmountFn) {
         unmountFn();
         return state.clearRefinements(attributeName);
       },
+
+      getWidgetState(fullState, { state }) {
+        const currentRefinements = state.getNumericRefinements(attributeName);
+        const equal = currentRefinements['='] && currentRefinements['='][0];
+        if (equal || equal === 0) {
+          return {
+            ...fullState,
+            numericRefinementList: {
+              ...fullState.numericRefinementList,
+              [attributeName]: `${currentRefinements['=']}`,
+            },
+          };
+        }
+
+        const lowerBound =
+          (currentRefinements['>='] && currentRefinements['>='][0]) || '';
+        const upperBound =
+          (currentRefinements['<='] && currentRefinements['<='][0]) || '';
+
+        if (lowerBound !== '' || upperBound !== '') {
+          if (
+            fullState.numericRefinementList &&
+            fullState.numericRefinementList[attributeName] ===
+              `${lowerBound}:${upperBound}`
```

Comment:
If one of the `lowerBound` or `upperBound` is `=== ''` then it will end like this:

* `xx:` or `:xx`

Is it expected?

Reply:
Yes